docs/src/HQMTool.md

```diff
@@ -0,0 +1,644 @@
+# HQMTool
```

Comment:
The docs probably need a fairly substantial reorganization to better highlight these features and make the docs easier to navigate. Also, some of it will need rewritten once we decide on what to do with demos (e.g. making them examples) and possibly writing some tutorials.